proto/src/determined/notebook/v1/notebook.proto

```diff
@@ -48,4 +49,6 @@ message Notebook {
   string exit_status = 13;
   // The associated job id.
   string job_id = 14;
+  // Workspace ID.
+  int32 workspace_id = 17;
```

Comment:
non blocking question: is there a reason to skip 15 and 16? Is that a proto thing?

Reply:
those are [taken](https://www.tixr.com/groups/laughfactoryhw/events/laugh-riot-55847) but are just not defined in order. I initially missed it too and had the compiler yell at me.